src/accordion/_base.scss

```diff
@@ -0,0 +1,42 @@
+/**
+   * @file Accordion
+   * @module Accordion
+   * @overview
+   * @@@ todo: standardize height transition, also make proportional to max-height/media queries
+ **/
+
+.accordion-list-item {
+   margin: ms(1) 0;
+ }
+
+.accordion-section {
+  .accordion-section-list {
+    overflow: hidden;
+    max-height: 0;
+    transition: max-height 0.3s ease-in-out;
+  }
+
+  .accordion-section-toggle {
+    cursor: pointer;
+
+    [data-icon="arrow"] {
+      transform: rotate(90deg);
+    }
+  }
+
+  &.is-open {
+    .accordion-section-list {
+      max-height: 1000px;
```

Comment:
This isn't ideal, but we don't have a good way to animate height at the moment.  These values would also have to be reset depending on the content for each accordion.